tests/unit_test/data_processor/action_serializer_test.py

```
@@ -0,0 +1,601 @@
+
+    for k in test_other_collection_keys:
+        assert k in other_collection_info_keys
+        assert other_collection_info[k].get('db_model') is not None
+
+    assert len(action_info_keys) == 17
+    assert len(other_collection_info_keys) == 1
+
+
+def test_is_action():
+    assert ActionSerializer.is_action(ActionType.http_action.value) == True
+    assert ActionSerializer.is_action(ActionType.two_stage_fallback.value) == True
```

Ruff

Avoid equality comparisons to `True`; use `if ...:` for truth checks

Replace comparison

(E712)